src/main/java/com/redhat/red/build/koji/model/xmlrpc/KojiTaskRequest.java

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright (C) 2015 Red Hat, Inc. (jcasey@redhat.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.redhat.red.build.koji.model.xmlrpc;
+
+import java.util.Collections;
+import java.util.List;
+
+public class KojiTaskRequest
+{
+    private List<Object> request;
+
+    public KojiTaskRequest( List<Object> request )
```

Comment:
We should also provide a default constructor so that the class can be deserialized from JSON.